composer.json

```
@@ -25,16 +25,21 @@
         "symfony/symfony": "^3.4"
     },
     "require-dev": {
+        "ezsystems/ezplatform-code-style": "^1.0@dev",
         "friendsofphp/php-cs-fixer": "^2.15",
         "phpunit/phpunit": "^8.3.5"
     },
     "scripts": {
         "test": "phpunit -v -c phpunit.xml",
-        "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
+        "fix-cs": "php-cs-fixer fix -v --show-progress=estimating",
+        "php-cs": "php-cs-fixer fix -v --diff --dry-run --show-progress=estimating"
```

```suggestion
        "check-cs": "php-cs-fixer fix -v --diff --dry-run --show-progress=estimating"
```

As mentioned in the https://github.com/ezsystems/ezplatform-core/pull/39#discussion_r654259967 , for consistency with other packages